libs/editor/egui_editor/src/input/mutation.rs

```diff
@@ -353,6 +350,9 @@ pub fn calc(
                     } else {
                         continue; // only process indented galleys
                     };
+                if !renumbering_processed_galleys.insert(galley_idx) {
```

Comment:
...now, renumber processing is handed by a separate set, with the original indentation processing set being referenced to determine whether to insert a tab